dietpi/dietpi-software

```diff
@@ -11148,18 +11149,18 @@ max-download-limit=0
 seed-ratio=0.1
 seed-time=0
 
-metalink-servers=$(Optimize_BitTorrent 2)
+# metalink-servers=$(Optimize_BitTorrent 2)
 allow-overwrite=false
 always-resume=true
 auto-file-renaming=false
 file-allocation=none
 
```

Comment:
The settings below were (from the old web UI) were actually meant to be used either|or. So either you limit download/upload speed or you don't. Like this, I guess the `0` will override the limit above.

`0` is default, leave choice to user to enable limits: https://aria2.github.io/manual/en/html/aria2c.html#cmdoption-max-overall-download-limit